requirements.txt

```diff
@@ -7,7 +7,7 @@ coreir>=2.0.123  # should be first so we get the latest version
 -e git://github.com/Kuree/karst.git#egg=karst
 -e git://github.com/joyliu37/BufferMapping#egg=buffer_mapping
 -e git+git://github.com/pyhdi/pyverilog.git#egg=pyverilog
--e git+https://github.com/StanfordAHA/lake#egg=lake
+-e git+https://github.com/StanfordAHA/lake@csv_names#egg=lake
```

Comment:
point back to master?

Reply:
done, CI will fail till csv_names merged into lake master.